### ISSUE TITLE
dataset 

### ISSUE THREAD

**Issue by zhjw0927** [huanglianghua/GlobalTrack#18]
assuming dataset storied in ./data
What do these data sets consist of? Training set, test set?
Can you explain the directory structure?
Thank you!
